cli/flox-rust-sdk/src/models/environment/remote_environment.rs

```diff
@@ -26,7 +80,11 @@ impl Environment for RemoteEnvironment {
         packages: Vec<String>,
         flox: &Flox,
     ) -> Result<InstallationAttempt, EnvironmentError2> {
-        todo!()
+        let result = self.inner.install(packages, flox).await?;
+        self.inner
+            .push(false)
+            .map_err(RemoteEnvironmentError::UpdateUpstream)?;
+        Ok(result)
```

Comment:
blocking (unless you want to followup): are you symlinking somewhere?

Reply:
the `ManagedEnv` will set a out link.. so activate should work already

Reply:
I haven't checked, but I'm expecting remote environments should create a `<system>.<name>` link, and managed should create `<system>.<name>.encode(<path of user dir>)`. If you're relying on `ManagedEnvironment`, is it giving it the wrong name?

Reply:
Can you also verify in a test?

Reply:
it creates a temporary link `<system>.<name>.encode(tempdir)`

Reply:
i mean yeah?
I guess the question is whether `--remote` will always affect the same out links in all processes or be a bit more isolated.. I see reason for both

Reply:
If you don't want to block, that's fine by me. I would probably link to `<system>.<name>` and add a test for it though. What are some downsides of using the same out link?

Reply:
changed it to use the same and added a test

still blovked on #621 and #620